src/SprykerEco/Client/Payone/Zed/PayoneStub.php

```diff
@@ -158,4 +160,16 @@ public function sendConsumerScoreRequest(QuoteTransfer $quoteTransfer): Consumer
     {
         return $this->zedStub->call('/payone/gateway/send-consumer-score-request', $quoteTransfer);
     }
+
+    /**
+     * @uses \SprykerEco\Yves\Payone\Controller\KlarnaController::getTokenAction()
```

Comment:
* It should point to a zed gateway controller.

Reply:
Thanks